fe-ionic/Jenkinsfile

```diff
@@ -14,7 +14,7 @@ node {
 
 library("ods-jenkins-shared-library@${sharedLibraryRef}")
 
-def ionicVersion = "5.4.16"
+def ionicVersion = "6.13.1"
```

Comment:
@bljubisic I'm not entirely sure whether this version refers to the Ionic [framework](https://ionicframework.com/docs/reference/release-notes) or just the CLI?

Reply:
I would say for Ionic framework, not just CLI